components/scg/src/scg-struct.js

```diff
@@ -234,6 +234,31 @@ function scgScStructTranslator(_editor, _sandbox) {
             /// --------------------
             var translateNodes = async function () {
                 var implFunc = async function (node) {
+                    if (node.text) {
+                        let linkAddrs = await scClient.getLinksByContents([node.text]);
+                        if (linkAddrs.length) {
+                            linkAddrs = linkAddrs[0];
+                            if (linkAddrs.length) {
+                                let link = linkAddrs[0];
+                                let scTemplate = new sc.ScTemplate();
+                                const ELEMENT = 'element';
+                                scTemplate.tripleWithRelation(
```

Comment:
Looks like you can use `result = await searchNodeByIdentifier(link, window.scKeynodes["nrel_system_identifier"]);` instead